flyio/caveats.go

```diff
@@ -314,3 +316,66 @@ func (c *NoAdminFeatures) Prohibits(a macaroon.Access) error {
 
 	return nil
 }
+
+// Commands is a list of command names allowed by this token.
+type Commands struct {
+	Commands []string `json:"commands"`
+}
```

Comment:
The original caveats from Thomas's implementation all have a bit more structure than is really necessary. I think this could be 
```suggestion
type Commands []string
```

Reply:
yup. a bit cleaner. latest commit doesnt wrap the array in a structure.